lib/chef/knife/vsphere_vm_list.rb

```diff
@@ -47,11 +56,16 @@ def print_vm(vm)
     puts "\t\t#{ui.color('State:', :magenta)} #{state}"
   end
 
+  def print_folder(folder)
+    puts "#{ui.color('Folder', :cyan)}: " + (folder.path[3..-1].map { |x| x[1] }.* '/')
+  end
+
   def run
     vim_connection
     base_folder = find_folder(get_config(:folder))
-    recurse =  get_config(:recursive)
+    only_folders = get_config(:onlyfolders)
+    recurse = only_folders ? true : get_config(:recursive)
```

Comment:
Hi, since we're dealing entirely with booleans and want to represent a priority, what would you think about using the boolean operators instead of the ternary operator?

```
recurse = only_folders || get_config(:recursive)
```

The ternary operator is a great way to make an `if/else` very concise, but I find that if we're dealing entirely in booleans that the regular operators read more English-like, and are therefore more expressive.